File: jqwik-starter-gradle/src/test/java/my/example/project/MyCalculatorExamples.java
package my.example.project;

import net.jqwik.api.*;
import org.assertj.core.api.*;

class MyCalculatorExamples {

	@Example
	void summingUpZeros() {
		int result = new MyCalculator().sum(0, 0, 0);
		Assertions.assertThat(result).isEqualTo(0);
	}

	@Example
	void summingUpTwoNumbers() {
		int result = new MyCalculator().sum(1, 41);
		Assertions.assertThat(result).isEqualTo(42);
	}
}

File: jqwik-starter-gradle-kotlin/src/test/kotlin/my/example/project/MyCalculatorExamples.kt
package my.example.project

import net.jqwik.api.Example
import org.assertj.core.api.Assertions

class MyCalculatorExamples {
    @Example
    fun summingUpZeros() {
        val result = MyCalculator().sum(listOf(0, 0, 0))
        Assertions.assertThat(result).isEqualTo(0)
    }

    @Example
    fun summingUpTwoNumbers() {
        val result = MyCalculator().sum(listOf(1, 41))
        Assertions.assertThat(result).isEqualTo(42)
    }
}